tests/data/netinfo/sample-ipaddrshow-output-metric

```diff
@@ -0,0 +1,12 @@
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    inet 127.0.0.1/8 scope host lo\       valid_lft forever preferred_lft forever
```

Comment:
Are the backslashes on lines 3 and 4 a typo?

Reply:
I just copied the `sample-ipaddrshow-output` file and added the metric part

Reply:
I see, nevermind. It doesn't matter what it says after `scope` since the regex ends with `.*`.